docs/UserGuide.md

```diff
@@ -144,6 +144,22 @@ Examples:
 * `delete warehouse 1` Removes the warehouse at index 1.
 * `delete supplier 12` Removes supplier at index 12 from the list of suppliers.
 
+### Deletes a product in a certain warehouse or supplier : `delete`
+
+Deletes a product entry no longer stored by a certain warehouse or sold by a specific supplier.
+
+Format: `delete TYPE INDEX pd/PRODUCT_NAME`
+
+* The PRODUCT_NAME must be identifiable and starts with alphanumeric character.
+* The product with the PRODUCT_NAME should be in the INDEX-th supplier/warehouse in the displayed list.
+
+Examples:
+
+* `delete warehouse 1 pd/Panadol` Removes product with the name _Panadol_ from
+the warehouse at index 1 of the list of warehouse.
+* `delete supplier 12 pd/Aspirin` Removes product with the name _Aspirin_
+the from supplier at index 12 from the list of suppliers.
```

Comment:
Thanks for spotting the type!

Reply:
> ld try to make the explana

May I know what do you mean by making the explanation consistent?

Reply:
Ahh oops! I was wondering if it would be possible to make the explanation for the examples in line 159 and 161 consistent? In line 159, it says "of the list of warehouse" and then in line 161 it says "from list of suppliers". Would it be possible to use the same format for both??

Reply:
I see, that's actually a typo also haha, thanks for pointing it out!